cg/store/crud/update.py

```diff
@@ -77,6 +77,11 @@ def update_sample_reads_illumina(self, internal_id: str, sequencer_type: Sequenc
         sample.reads = total_reads_for_sample
         self.session.commit()
 
+    def update_sample_reads(self, internal_id: str, reads: int):
```

Comment:
Will this method work if a sample is spread over several runs? Would we not want to add the reads to any prior ones?

Reply:
That is a good point. We will implement that when we have pooled samples

Reply:
implementing +=